x/permission/keeper/keeper.go

```diff
@@ -69,7 +69,7 @@ func (k Keeper) GetGovernanceKey(ctx sdk.Context) (multisig.LegacyAminoPubKey, b
 func (k Keeper) GetRole(ctx sdk.Context, address sdk.AccAddress) exported.Role {
 	govAccount, ok := k.getGovAccount(ctx, address)
 	if !ok {
-		return exported.ROLE_UNSPECIFIED
+		return exported.ROLE_UNRESTRICTED
```

Comment:
Why not return Unspecified if it doesn't exist?

Reply:
same question here.

Reply:
I need to be able between the case where it is missing and the case where it is purposefully set to be unrestricted to enforce the option